docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -3,218 +3,1550 @@
   items:
```

Comment:
Maybe have two top level nodes: Errors and Warnings. Then nest the individual messages underneath, omitting either "Compiler error" or "Compiler warning" from the displayed text. As is, it's one very long list, and a lot of repeated words.

Reply:
Great suggestion!